assets/scss/core/_base.scss

```diff
@@ -236,6 +236,11 @@ input[type="tel"] {
   display: none;
 }
 
+.form-message {
+  margin-bottom: $dp-spaces--lv2;
+  font-family: $dp-font-family;
```

Comment:
Hey @GusBaamonde, we did not see it, but without this change, span is rendered with serif font. Do you think that it is fine? should apply it in another place?

Reply:
@GusBaamonde commented me that @santiagopaolucci said that we should use `$dp-font-family-proximanova` in all public pages.

I will check if "validation errors" is the only missing, in that case, I will fix it, if not, I will create an issue.

Reply:
The fix is ready! Resolving the conversation...